Explorer/Assets/DCL/SDKComponents/SceneUI/Components/UITransformComponent.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Arch.Core;
+using DCL.Optimization.Pools;
+using System;
+using System.Collections.Generic;
+using UnityEngine.Pool;
+using UnityEngine.UIElements;
+
+namespace DCL.SDKComponents.SceneUI.Components
+{
+    public struct UITransformComponent : IPoolableComponentProvider<VisualElement>
+    {
+        public VisualElement Transform;
+        public EntityReference Parent;
+        public int RightOf;
```

Comment:
You can get `RightOf` from the SDK component directly so you don't need to duplicate the field, can't you?